src/controllers/users_controller.js

```diff
@@ -76,6 +76,11 @@ module.exports.changeUserInfo = (
       error: 'Password must be provided.',
     });
   }
+  if (!name && !newPassword) {
```

Comment:
are we testing for cases when new username and new password is not provided

Reply:
yup, there was a new test case for this :)